solutions/13/script.js

```diff
@@ -0,0 +1,31 @@
+
+const ul = document.querySelector('.authors'); // Get the list where we will place our authors
+const url = 'https://randomuser.me/api/?results=5';
+
+function createNode(element) {
+    return document.createElement(element);
+}
+
+function append(parent, el) {
+  return parent.appendChild(el);
+}
+
+fetch(url)
+  .then(response => response.json())
+  .then(data => {
+    let authors = data.results;
+    return authors.map(function(author) {
+        const { first, last } = author.name;
+      let li = createNode('li'),
```

Comment:
```
 let li = createNode('li'),
          img = createNode('img'),
          span = createNode('span');
```
Generally it's ok, but i haven't seen that in real project

Reply:
Changed 👍